### ISSUE TITLE
Explicit statement on limited uses

### ISSUE THREAD

**Issue by GammaGames** [afranka70/AbOutIssues#18]
# AbOut Version
326s17
# Submitted by
@afranka70 
# Description
I should be noted in the SRS that is application assumes that it will be used only by a small, congenial, and security-conscious group, as any CAS user could potentially become an AbOut user.

**Comment by LoganSWarner** [comment]
I don't think this is valid for the long-term plan and I *don't* think we should be relying on the people handling security - it's a FERPA violation waiting to happen with a flat user hierarchy.
